tests/queries/0_stateless/01591_window_functions.sql

```diff
@@ -1,3 +1,5 @@
+-- Tags: no-random-settings
```

Comment:
some timeouts during flaky checks are acceptable. we are better to not add this tag.

Reply:
Since priority is now a randomized setting, 01591_window_functions (and maybe some other tests) almost always time out when the priority is low (value of 3 or higher). 
If adding the tag `no-random-settings` is not a good idea, should we add the tag `long`?

Reply:
adding `long` probably makes sense

Reply:
done